docs/content/latest/integrations/metabase.md

```diff
@@ -46,7 +48,7 @@ orders.sql  products.sql  reviews.sql  users.sql
 
 ### Connect to YugabyteDB using ysqlsh
 
-You can do this as shown below.
+You can do this by executing the following:
```

Comment:
```suggestion
Run the following command to connect to YugabyteDB using the YSQL shell:
```

Reply:
Changed in DocMinorFixesForPR9105 branch